recipes/zoltan/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+mpi:
+  - mpich
+#  - openmpi
```

Comment:
I don't think you need this file since you are just using ``mpich``. So, you can just put explicit ``mpich`` on the recipe